pkg/cmd/infra/router/router.go

```diff
@@ -83,6 +95,50 @@ func (o *RouterSelection) RouteSelectionFunc() controller.RouteHostFunc {
 	}
 }
 
+func (o *RouterSelection) AdmissionCheck(route *routeapi.Route) error {
+	if len(route.Spec.Host) < 1 {
+		return nil
+	}
+
+	if hostInDomainList(route.Spec.Host, o.BlacklistedDomains) {
+		glog.V(4).Infof("host %s in list of denied domains", route.Spec.Host)
```

Comment:
Should we put quotes around the hostnames?  Or is that not the pattern elsewhere?